clients/python/client/osparc/_files_api.py

```diff
@@ -91,10 +91,10 @@ async def upload_file_async(self, file: Union[str, Path]) -> File:
 
         tasks: list = []
         async with AsyncHttpClient(
-            base_url=httpx.URL(self.api_client.configuration.host),
             request_type="post",
             url=links.abort_upload,
             auth=self._auth,
+            follow_redirects=True,
```

Comment:
for the pre-signed links, right?

Reply:
This was because the `abort` and `complete` links I got back from the Api server were using http instead of https. This is very strange, because they are generated using starlette.Request. But in any case I will switch to using relative links soon (as discussed) so this shouldn't be an issue